app/src/main/kotlin/com/hippo/ehviewer/download/DownloadManager.kt

```diff
@@ -835,5 +835,5 @@ var downloadLocation: UniFile
     }
 
 val DownloadInfo.downloadDir get() = dirname?.let { downloadLocation / it }
-val DownloadInfo.archiveFile get() = downloadDir?.findFile("$gid.cbz")
+val DownloadInfo.archiveFile get() = downloadDir?.run { findFile("$gid.cbz") ?: findFile("$gid.zip") }
```

Comment:
Why zip? We may get 7z from torrent.

Reply:
Archives from torrents are not guaranteed to have the exact same content as the gallery (may have more as some of the images can't be uploaded, or less in the case of outdated torrents), so that would be a more complicated case.